dashboard/modules/job/job_manager.py

```diff
@@ -393,7 +393,10 @@ def __init__(self, gcs_aio_client: GcsAioClient, logs_dir: str):
         self._gcs_address = gcs_aio_client._channel._gcs_address
         self._log_client = JobLogStorageClient()
         self._supervisor_actor_cls = ray.remote(JobSupervisor)
-        self.event_logger = get_event_logger(Event.SourceType.JOBS, logs_dir)
+        try:
+            self.event_logger = get_event_logger(Event.SourceType.JOBS, logs_dir)
+        except Exception:
+            self.event_logger = None
```

Comment:
What's this try-except for? I guess this will just cause an exception down the road whenever `_monitor_job` is run and it tries to call `None.info(event_log)`, but why is that better?